3rdparty/CMakeLists.txt

```diff
@@ -389,7 +368,13 @@ add_library(3rdparty::pugixml ALIAS pugixml)
 add_library(3rdparty::yaml-cpp ALIAS yaml-cpp)
 add_library(3rdparty::xxhash ALIAS xxhash)
 add_library(3rdparty::hidapi ALIAS 3rdparty_hidapi)
-add_library(3rdparty::libpng ALIAS ${LIBPNG_TARGET})
+
+if(USE_SYSTEM_LIBPNG)
+	add_library(3rdparty::libpng ALIAS 3rdparty_system_libpng)
```

Comment:
The old approach of
`add_library(3rdparty::libpng ALIAS ${LIBPNG_TARGET})`
Was cleaner IMO